connections/debug/debug_connection.py

```diff
@@ -89,7 +90,7 @@ def bulk_sensor_mock_random(self) -> bytearray:
                                          random.uniform(0, 1e6),
                                          random.uniform(49.260565, 49.263859),
                                          random.uniform(-123.250990, -123.246956),
-                                         random.randint(0, 100))
+                                         random.randint(0, 0x09))
```

Comment:
Guessing this is mocking the state, in which case it'd be nice to have this not be a magic number

Reply:
Would you prefer a global containing the range of the STATE_IDS, extracted from the dictionary itself? Not sure what you mean by magic number here though, considering it's a randomized selection of inputs for the bulk debug packet